dashjoin-rdf4j/src/test/java/org/dashjoin/service/RDF4JTest.java

```diff
@@ -54,9 +56,41 @@ public void testGetTables() throws Exception {
     Assertions.assertTrue(db.tables().contains("dj/junit/http:%2F%2Fex.org%2FU"));
   }
 
-  @Override
-  @Test
-  public void testPath() throws Exception {
-    // SPARQL does not support tracking property paths
-  }
+//  @Override

```

Comment:
I think we need the empty override. Otherwise, I get a SPARQL parse error since OpenCypher query from DBTest is fed. Note that the RDF4JTest is not part of the CICD test on GitHub yet